packages/astro/src/core/config.ts

```diff
@@ -12,6 +13,9 @@ import loadTypeScript from '@proload/plugin-tsm';
 import postcssrc from 'postcss-load-config';
 import { arraify, isObject } from './util.js';
 import { appendForwardSlash, trimSlashes } from './path.js';
+// These two imports make astroMarkdownOptions work
```

Comment:
wahhh??? Would love an explanation why here, for future readers

Reply:
Check out the new commit, I added a longer (not sure if better) explanation